.github/workflows/update-viablestrict.yml

```diff
@@ -0,0 +1,45 @@
+name: Update viable/strict
+
+on:
+  schedule:
+    - cron: 17,47 * * * *
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  do_update_viablestrict:
+    runs-on: ubuntu-20.04
```

Comment:
@seemethere is this a good runner for these? I think these are okay, but maybe one of the premium runners?

Reply:
Since this doesn't run very often I think these should be fine to use